fuel-core/src/state/rocks_db.rs

```diff
@@ -221,6 +221,8 @@ impl KeyValueStore for RocksDb {
             .iterator_cf_opt(&self.cf(column), opts, iter_mode)
             .map(|item| {
                 item.map(|(key, value)| {
+                    // TODO: Avoid copy of the slices into vector.
```

Comment:
We could also adjust the API to make use of boxed slices instead of vec 🤔

Reply:
We can, the question do we want to add complexity to database API=) But it will force us to work with raw slice where it is possible=)